src/desktop/src/ui/views/onboarding/Login.js

```diff
@@ -105,10 +111,18 @@ class Login extends React.Component {
      * @returns {undefined}
      */
     setupAccount = async () => {
-        const { wallet, currency, currentAccountName, currentAccountMeta } = this.props;
-
-        const accountName = wallet.addingAdditionalAccount ? wallet.additionalAccountName : currentAccountName;
-        const accountMeta = wallet.addingAdditionalAccount ? wallet.additionalAccountMeta : currentAccountMeta;
+        const {
+            wallet,
```

Comment:
This could be removed. It's only used for password. I suggest adding password to mapStateToProps

Reply:
Fixed with https://github.com/iotaledger/trinity-wallet/pull/587/commits/9977967a2c29a396365dec9d974a0315ae537df9